quizzes/graphql/mutations/classes.py

```diff
@@ -0,0 +1,74 @@
+import graphene
+import jwt
+import time
+
+from decouple import config
+from quizzes.models import Class, Teacher
+
+'''
+CreateClass
+'''
+class CreateClass(graphene.Mutation):
+    '''
+    The `Arguments` nested class is how GraphQL knows what arguments
+    are required when making a mutation/POST request.
+    
+    The `ok` prop is required by GraphQL. This is not a Django Attribute
+    The `new_class` prop is calling the `ClassMutation` class right below
+    our `CreateClass` class. We then call the `mutate()` method, denoted with
+    a `@staticmethod` decorator, that will then use the value of
+    `ClassMutation.ClassName` and pass that to the `Class.objects.create()`
+    method saved on the `new_class` variable inside of the
+    `mutate()` method below
+    EXAMPLE GRAPHQL MUTATION
+    ------------------------
+    
+    mutation {
+        createClass(ClassName:"Test Class Creation 1") {
+            newClass {
+                ClassName
+            }
+            ok
+        }
+    }
+    '''
+    class Arguments:
+        ClassName  = graphene.String()
+        enc_jwt    = graphene.String()
+
+    new_class = graphene.Field(lambda: ClassMutation)
+
+    @staticmethod
+    def mutate(self, info, ClassName, enc_jwt):
+        secret    = config('SECRET_KEY')
+        algorithm = 'HS256'
+        dec_jwt   = jwt.decode(enc_jwt, secret, algorithms=[ algorithm ])
+
+        # `user` variable needs to be changed to use the ID given by the JWT
+        # for now JWT does NOT return a userID
+        user = Teacher.objects.get(TeacherEmail=dec_jwt[ 'sub' ][ 'email' ])
+        
+        # if token is expired return expiration error to user
+        if dec_jwt[ 'exp' ] < time.time():
+            return print('\n\nTOKEN EXPIRED\nRETURN ERROR TO CLIENT\n\n')
+        
```

Comment:
Can these three `print` errors be changed to return GraphQL errors for the client to use?